test/onnx/onnx_test.cpp

```diff
@@ -3782,6 +3782,63 @@ TEST_CASE(tanh_test)
     EXPECT(p == prog);
 }
 
+TEST_CASE(thresholdedrelu_default_test)
```

Comment:
Just wondering, 
All three tests different in alpha value or data type. I was wondering if there is thing like test parametrization. For example as in `pytest`.